docs/3to4.md

```diff
@@ -97,44 +97,44 @@
     - getLocalHostQuietly
 
 - Protocol class
-  - All String variables representing to Sentinel, Cluster and PubSub sub-commands are removed.
+  - All String variables representing to Sentinel, Cluster and PubSub sub-commands have been removed.
 
-- Sentinel methods are removed from Jedis class and put in new Sentinel class.
+- Sentinel methods were removed from Jedis class and can be found in the new Sentinel class.
 
-- Transaction inside Pipeline is removed. E.g. multi(), exec() and discard() methods are removed from Pipeline.
+- Transaction inside Pipeline were removed. E.g. multi(), exec() and discard() methods have been removed from Pipeline.
 
-- execGetResponse() method is removed from Transaction.
+- execGetResponse() method was removed from Transaction.
 
-- JedisClusterHashTagUtil class is renamed to JedisClusterHashTag.
+- JedisClusterHashTagUtil class was renamed to JedisClusterHashTag.
 
-- KeyMergeUtil class is removed.
+- KeyMergeUtil class was removed.
 
 - Jedis.aclLog(String) and Jedis.aclLog(byte[]) methods are removed.<br>
-  Jedis.aclLogReset() method is added to serve the purpose.
+  Jedis.aclLogReset() method was added to serve the purpose.
 
-- Jedis.debug(DebugParams) method is removed.
+- Jedis.debug(DebugParams) method was removed.
 
-- Jedis.sync() method is removed.
+- Jedis.sync() method was removed.
 
 - Jedis.configSetBinary(byte[], byte[]) method. Use Jedis.configSet(byte[], byte[]).
 
-- Jedis.pubsubNumSub(String...) method now returns `Map<String, Long>` instead of `Map<String, String>`.
+- Jedis.pubsubNumSub(String...) method returns `Map<String, Long>` instead of `Map<String, String>`.
 
-- Jedis.scriptExists(byte[]) now returns `Boolean` instead of `Long`.
+- Jedis.scriptExists(byte[]) returns `Boolean` instead of `Long`.
 
-- Jedis.scriptExists(byte[]...) now returns `List<Boolean>` instead of `List<Long>`.
+- Jedis.scriptExists(byte[]...) returns `List<Boolean>` instead of `List<Long>`.
 
-- `watch` and `unwatch` methods of Transaction class is not supported within MULTI (after multi() method). It can be used before MULTI.
+- `watch` and `unwatch` methods of Transaction class is unsupported within MULTI (after multi() method). It can be used before MULTI.
 
-- Following cases would throw IllegalStateException instead of JedisDataException.
+- The following cases would throw an IllegalStateException instead of a JedisDataException.
```

Comment:
Each of the following cases would throw an IllegalStateException instead of a JedisDataException.